src/Form/Input.js

```diff
@@ -277,3 +278,5 @@ export default class Input extends Component {
     );
   }
 }
+
+export default createEasyInput(Input);
```

Comment:
do you intend for `Input`s to be overridable? e.g.
```js
<Input name="my_field" value={some_function(values.my_field)} />
```

looks like the HOC will take precedence

Reply:
Yeah - props are spread last so any auto-bound prop can be overridden. Seems like the right behavior but do you have a case in mind where we wouldn't want that? 
```diff
  if (!state) {
    return <PureInput name={name} Component={Component} {...props} />
  }

   return (
    <PureInput
      name={name}
      value={state.values[name] || ''}
      error={state.errors[name]}
      onChange={state.onChange}
      onBlur={state.onBlur}
      onFocus={state.onFocus}
      Component={Component}
+    {...props}
    />
  );
```

Reply:
no im just dumb, i missed the spread